cmd/noms/noms.go

```diff
@@ -31,6 +33,13 @@ var commands = []*util.Command{
 	nomsVersion,
 }
 
+type kCommandHandler func(input string) (exitCode int)
```

Comment:
yeah was wondering if this was gonna look too hungarian. Was planning on removing once the kingpin stuff no longer needed to be differentiated, but I think this is a simple enough file to not need that.